Automattic/Gravatar-SDK-iOS
# Quick Editor UIKit

Closes #424 

### Description

Adding a `QuickEditorPresenter` to be able to present the quick editor from a UIKit view controller.

### Testing Steps

- Run the UIKit demo app.
- Go to `Quick Editor` demo screen.
  - Check that everything works as expected.
